frontend/src/pages/ModelTester/Dashboard2/Dashboard2.jsx

```diff
@@ -179,7 +190,7 @@ const Dashboard2 = () => {
       )}
 
       <div className="chart-section">
-        {isChartVisible && Object.keys(graphData).length > 0 && (
```

Comment:
Why are we removing is chart visible here

Reply:
We are now displaying an empty graph instead of nothing as it looks better and gives the user an idea of possible outputs